packages/apps/plugins/plugin-space/src/util.tsx

```diff
@@ -41,7 +41,7 @@ export const isSpace = (data: unknown): data is Space =>
     : false;
 
 export const getSpaceDisplayName = (space: Space): string | [string, { ns: string }] => {
-  return (space.properties.name?.length ?? 0) > 0
+  return space.state.get() === SpaceState.READY && (space.properties.name?.length ?? 0) > 0
```

Comment:
is the space state not handled by the second part of this ternary? my understanding is that the properties should be cached so that they're accessible even if the space is closed

Reply:
I deleted cached properties, they were broken for 2 months now, and we do not plan to fix them

Reply:
Now properties is available only if state is `READY`

Reply:
we need to discuss how we're going to support an equivalent UX to the one cached properties enabled then